Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
And kudos for that!!!
On Wed, 1 Jun 2016 at 19:18, Scott Lahteine <notifications@github.com>
wrote:

> In Marlin/ultralcd.cpp
> <https://github.com/MarlinFirmware/Marlin/pull/2105#discussion_r65342515>:
>
> > @@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
> >  #ifdef HAS_LCD_CONTRAST
> >    static void lcd_set_contrast() {
> >      if (encoderPosition != 0) {
> > -      lcd_contrast -= encoderPosition;
> > -      lcd_contrast &= 0x3F;
> > +      #ifdef U8GLIB_LM6059_AF
> > +        lcd_contrast += encoderPosition;
>
> I also fled the scene for a while, for the same reason. But I'm happy to
> report that Richard Wackerbarth was dismissed from the project a couple of
> months ago. We are getting along very well these days, with good
> camaraderie and a much more friendly environment.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MarlinFirmware/Marlin/pull/2105/files/082ed3beedfaffaa7fb8b8e2ae4fa2f0f9f8aca4#r65342515>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGcPb_JraEmiAmRz9kz4qlOocQlZhVQsks5qHWqhgaJpZM4Ed7jG>
> .
>